package.json

```diff
@@ -24,7 +24,7 @@
   ],
   "scripts": {
     "test": "bundlesize && jest --coverage --no-cache",
-    "build": "npm run bundle && npm run minify",
+    "build": "npm run bundle && npm run minify && cp hyperapp.js.flow dist/",
```

Comment:
actually no it won't, flow looks for the file named hyperapp.js.flow next to the hyperapp.js ...  infact it may need to be named hyerapp.min.js.flow for it to work seamlessly.